research/flamby/fed_isic2019/fenda/client.py

```diff
@@ -33,6 +33,7 @@ def __init__(
         loss_meter_type: LossMeterType = LossMeterType.AVERAGE,
         metric_meter_type: MetricMeterType = MetricMeterType.ACCUMULATION,
         checkpointer: Optional[TorchCheckpointer] = None,
+        type_run: str = "cos_sim",
```

Comment:
I'd say that we don't want the default to be using a "cos_sim" loss, but rather the vanilla FENDA approach. That is, no auxiliary losses attached.